draft-ietf-rats-corim.md

```diff
@@ -2367,14 +2373,41 @@ Endorsements are added to the ACS if the Endorsement condition is satisifed by t
+The Endorsed Values Triple contains a single `environment-map`, which means it can only match against one environment in the ACS; it can only add a single Endorsement ECT; and it uses the same enviromnent for `condition` and `addition`.
+
+After transformation into an `ev` entry, the processing steps of both triples are the same, as described below.
+Each `ev` entry is processed independently of other `ev`s.
+
+The Verifier SHALL find all sets of ACS-ECTs which match all the `ev`.`condition`s, where the ACS-ECT `cmtype` contains either `evidence`, `reference-values`, or `endorsements`.
+For each match ({{sec-match-condition-ect}}), all the `ev`.`addition` ECTs are added to the ACS.
+
+Note that some condition values are able to match against multiple ACS-ECTs, or sets of ACS-ECTs.
+If there are multiple matches then each match is processed independently from the others.
+
+##### Copying instance and/or group fields from a condition {#sec-comid-instance-group-copy}
+
+A CoRIM author may need to create a conditional endorsement which applies to all all measurements which have the same `class` field within their `environment-map`, regardless of their `instance` and/or `group` fields.
+The `slot-bind-type` and `slot-ref-type` options in these fields can be used to achieve this.
+
+In the simplest case, the CoRIM sets the `enviroment-map`.`instance` field of a `stateful-environment-record` within the triple to hold an `slot-bind-type` and the `enviroment-map`.`instance` field of an `endorsed-triple-record` to hold an `slot-ref-type`.
+Both instance types contain an integer name for the slot variable which is bound by the `slot-bind-type` and referenced by the `slot-ref-type`.
 
-#### Processing Conditional Endorsements {#sec-process-cond-end}
+Within each triple, each slot variable can only be bound to one value, so if there are multiple `environment-map`s using `slot-bind-type` then subsequent uses introduce a constraint that the value to bind must be equal to the value already bound to the slot.
 
-Conditional Endorsement Triples are transformed into an internal representation based on `ev`.
-Conditional endorsements have the same processing steps as shown in ({{sec-process-end}}).
+After successfully matching a `stateful-environment-record` containing an `slot-bind-type` against an ACS entry, the verifier SHALL copy the instance value from that ACS entry to the corresponding slot variable.
+If the matching ACS entry does not include an instance then the slot variable is marked as bound to the empty value.
+
+When adding a conditional endorsement whose `enviroment-map`.`instance` field is an `slot-ref-type` to the ACS, and that variable is bound to a non-empty value, then the verifier SHALL set the `enviroment-map`.`instance` field from the contents of the corresponding slot variable.
```

Comment:
`enviroment-map`.`instance` field from the contents of the corresponding slot variable to where ??